lib/Net/SSLeay.pod

```diff
@@ -995,52 +995,101 @@ The following new functions behave in perlish way:
                                     # Performs SSL_write, but automatically
                                     # figures out the size of $foo
 
-=head3 Low level API: Version related functions
+=head3 Low level API: Version and library information related functions
 
 =over
 
-=item * SSLeay
+=item * OpenSSL_version_num and SSLeay
 
-B<COMPATIBILITY:> not available in Net-SSLeay-1.42 and before
+B<COMPATIBILITY:> SSLeay() is not available in Net-SSLeay-1.42 and
+before. SSLeay() was made an alias of OpenSSL_version_num() in OpenSSL
+1.1.0 and LibreSSL 2.7.0.
+
+B<COMPATIBILITY:> OpenSSL_version_num() requires at least
+Net-SSLeay-1.82 with OpenSSL 1.1.0, or Net-SSLeay-1.88 with LibreSSL
+2.7.0.
 
-Gives version number (numeric) of underlaying openssl library.
+Both functions return OPENSSL_VERSION_NUMBER constant (numeric) as
+defined by the underlaying OpenSSL or LibreSSL library.
 
  my $ver_number = Net::SSLeay::SSLeay();
- # returns: the number identifying the openssl release
- #
+or
+ my $ver_number = Net::SSLeay::OpenSSL_version_num();
+ # returns: OPENSSL_VERSION_NUMBER constant
+
+ OpenSSL version numbering is:
+
  # 0x00903100 => openssl-0.9.3
  # 0x00904100 => openssl-0.9.4
  # 0x00905100 => openssl-0.9.5
  # 0x0090600f => openssl-0.9.6
  # 0x0090601f => openssl-0.9.6a
- # 0x0090602f => openssl-0.9.6b
  # ...
  # 0x009060df => openssl-0.9.6m
  # 0x0090700f => openssl-0.9.7
  # 0x0090701f => openssl-0.9.7a
- # 0x0090702f => openssl-0.9.7b
  # ...
  # 0x009070df => openssl-0.9.7m
  # 0x0090800f => openssl-0.9.8
  # 0x0090801f => openssl-0.9.8a
- # 0x0090802f => openssl-0.9.8b
  # ...
- # 0x0090814f => openssl-0.9.8t
+ # 0x0090821f => openssl-0.9.8zh
  # 0x1000000f => openssl-1.0.0
- # 0x1000004f => openssl-1.0.0d
- # 0x1000007f => openssl-1.0.0g
+ # ...
+ # 0x1000014f => openssl-1.0.0t
+ # 0x1000100f => openssl-1.0.1
+ # ...
+ # 0x1000115f => openssl-1.0.1u
+ # 0x1000200f => openssl-1.0.2
+ # ...
+ # 0x1000215f => openssl-1.0.2u
+ # 0x1010000f => openssl-1.1.0
+ # ...
+ # 0x101000cf => openssl-1.1.0l
+ # 0x1010100f => openssl-1.1.1
+ # ...
+ # 0x101010df => openssl-1.1.1m
+ # 0x30000000 => openssl-3.0.0
+ # 0x30000010 => openssl-3.0.1
+
+ Note that OpenSSL 3.0.0 and later do not set the status nibble in the
+ least significant octet to f.
+
+ LibreSSL returns 0x20000000 always:
+
+ # 0x20000000 => libressl-2.2.1
+ # ...
+ # 0x20000000 => libressl-3.4.2
 
-You can use it like this:
+You can use the version number like this when you know that the
+underlaying library is OpenSSL:
```

Comment:
nit: `underlaying` -> `underlying`

Reply:
Fixed. Also fixed another instance elsewhere on the page.